base/deprecated.jl

```diff
@@ -374,6 +374,10 @@ end
 
 @deprecate ntuple(n::Integer, f::Function) ntuple(f, n)
 
+# 8856
+export assert
```

Comment:
The export is probably not necessary here since the `@deprecate` macro automatically exports.